src/toM3u8.js

```diff
@@ -246,10 +246,11 @@ export const organizeAudioPlaylists = (playlists, sidxMapping = {}, isAudioOnly
 export const organizeVttPlaylists = (playlists, sidxMapping = {}) => {
   return playlists.reduce((a, playlist) => {
     const label = playlist.attributes.label || playlist.attributes.lang || 'text';
+    const language = playlist.attributes.lang || playlist.attributes.label || 'en';
 
     if (!a[label]) {
       a[label] = {
-        language: label,
```

Comment:
oof, seems like a huge oversight of us not to have it set to the language, heh. Good catch!